brainglobe_atlasapi/atlas_generation/atlas_scripts/humanatlas.py

```diff
@@ -8,6 +8,8 @@
 import treelib
 import urllib3
 from allensdk.core.structure_tree import StructureTree
+
+### I had to install manually
```

Comment:
Again, worth adding to the dependencies. If it is very big or hard-to-install, it can be an optional dependency like `allensdk`.